srcpkgs/calamares/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'calamares'
+pkgname=calamares
+version=3.2.44.2
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release
+                -DCMAKE_INSTALL_PREFIX=/usr
+                -DCMAKE_INSTALL_LIBDIR=lib
+                -DWITH_PYTHONQT:BOOL=ON
+                -DINSTALL_CONFIG:BOOL=ON
+                -DINSTALL_POLKIT:BOOL=ON
+                -DBoost_NO_BOOST_CMAKE=ON
+                -DSKIP_MODULES='webview interactiveterminal services-systemd dummyprocess dummypython dummycpp dummypythonqt'"
+hostmakedepends="extra-cmake-modules boost-python3-1.72 qt5-devel python3-devel kpmcore-devel
+                gettext ckbcomp kcoreaddons kdelibs4support qt5-svg-devel yaml-cpp-devel
+                boost-python3-1.72 python3-QtPy pkg-config libpwquality qt5-webengine
+                AppStream-qt python3-yaml python3-jsonschema"
+makedepends="python3-devel python3-jsonschema python3-yaml boost-devel extra-cmake-modules
+            qt5-tools qt5-translations git boost boost-python3-1.72 python3-QtPy pkg-config
+            libpwquality qt5-webengine kde5 AppStream-qt plasma-framework-devel"
+depends="kconfig kcoreaddons kiconthemes ki18n kio ksolid yaml-cpp kpmcore boost ckbcomp
+        hardinfo qt5-svg polkit-qt5 gtk-update-icon-cache plasma-framework qt5-xmlpatterns
+        squashfs-tools libpwquality AppStream-qt icu"
```

Comment:
A lot of the *depends are messed up. `kde5`, `boost` are wrong in every occurrence. `git`, `extra-cmake-modules` and `pkg-config` should it only be in hostmakedepends

`boost-python*` in hostdeps is most likely wrong, same with `qt5-webengine` also the `-devel` packages in host should most likely be in `makedepends`